core/server/data/migration/006/index.js

```diff
@@ -0,0 +1 @@
+module.exports = [];
```

Comment:
Sorry, my mistake, I thought I was looking at the [fixtures array](https://github.com/TryGhost/Ghost/pull/6877/files#diff-3bad0b11966c337a815984c3fc13e69bR1)